packages/python-flake8/python-flake8.spec

```diff
@@ -68,12 +65,15 @@ set -ex
 %files -n %{?scl_prefix}python%{python3_pkgversion}-%{pypi_name}
 %license LICENSE
 %doc README.rst tests/fixtures/config_files/README.rst
-%{_bindir}/flake8
+%exclude %{_bindir}/flake8
```

Comment:
Looks like this is a dependency for `python-galaxy-importer`. Are you sure it only needs it as a library and not as an executable?

Reply:
Checking with the galaxy team...

Reply:
Looks like it is: https://github.com/ansible/galaxy-importer/blob/3315bb8aa76285aa5fb1623440f3f290fda4c89d/galaxy_importer/loaders/content.py#L129-L145

This may not be the only instance of it, just one that came to mind. Please check this for all other bins.